tenant_schemas/middleware.py

```diff
@@ -61,7 +62,8 @@ def process_request(self, request):
             )
 
         request.tenant = tenant
-        connection.set_tenant(request.tenant)
+        for conn in connections:
+            connections[conn].set_tenant(request.tenant)
```

Comment:
For a single request - it will only go to 1 db right? So ideally this should figure out which connection to setup the tenant on - instead of loopoing